autogen/cluster.tf

```diff
@@ -181,6 +181,10 @@ resource "google_container_cluster" "primary" {
       state    = database_encryption.value.state
     }
   }
+
+  workload_identity_config {
+    identity_namespace = var.identity_namespace
```

Comment:
What happens if `var.identity_namespace` is `""` or `null`? Will this block existing raise errors?

Reply:
If it is null, it raises a plan error:

> Error: "workload_identity_config.0.identity_namespace": required field is not set

If it is "", it creates a cluster without identity_namespace

>    ~ workload_identity_config {
>           - identity_namespace = "project.svc.id.goog" -> null
>         }
>

Reply:
Ok, let's use `""` as the default then.

Reply:
This fails when you have a pre-existing cluster made without this changeset. It tries to change the cluster definition to add an empty block `workload_identity_config {}`, with the provider failing: `Error: googleapi: Error 400: Must specify a field to update., badRequest`. This might be better served by a dynamic terraform block, to prevent the block from being defined at all if var.identity_namespace is empty.